release_notes/44-release-notes.adoc

```diff
@@ -0,0 +1,572 @@
+=== Integration with {cloud-redhat-com} and Paladin Cloud to discover unsecured clusters
+
+In {product-title-short} 4.4, you can now discover new clusters that are not protected by {product-title-short} through integration with {cloud-redhat-com} and Paladin Cloud. This feature provides a list of clusters in your Openshift environment or on cloud platforms such as Amazon Elastic Kubernetes Service (Amazon EKS), Google Kubernetes Engine (Google GKE) and Microsoft Azure Kubernetes Service (Microsoft AKS) and provides a discovery mechanism to improve the security of your organization.
+
+Paladin Cloud currently offers a free trial to help you discover how Paladin Cloud works with {product-title-short} to secure your {osp} and Kubernetes deployments in the cloud. 
+
+For more information about the free trial version, visit link:https://paladincloud.io/free-trial/[Paladin Cloud Free Trial].
+
+[id="ebpf-co-re-collection-method-enabled-by-default_{context}"]
+=== eBPF CO-RE collection method enabled by default
+
+{product-title-short} 4.4 introduces a default runtime collection method based on eBPF compile once-run everywhere (CO-RE), which becomes the default method starting with this release unless it is explicitly overridden in the configuration of your secured cluster. 
+
+The CO-RE approach is a modern method for creating portable eBPF applications that ensures compatibility across different kernel versions and configurations without requiring changes or compilation of runtime source code on the target machine. When you upgrade to the new version, the migration from eBPF to eBPF CO-RE is automatic.
+
+The following are some of the advantages of this new collection method:
+
+* You can run {product-title-short} secured clusters on a larger number of Linux operating systems that are not currently included in the support package.
+* If you are offline, you no longer need to update the Collector support packages.
+By avoiding issues related to eBPF probe retrieval, such as the risk of losing the network connection or if the probe is not present, you can update a cluster smoothly.
+
+[id="bring-your-own-database-for-central-database-is-now-generally-available_{context}"]
+=== Bring your own database for Central database is now generally available
+
+With {product-title-short} 4.4, you can now use your own PostgreSQL-compatible database for the {product-title-short} Central database, giving you the choice to deploy it either within or outside the cluster. Whether on bare metal, virtual machines or as a cloud-hosted service, you can tailor the deployment to your specific requirements. To ensure optimal performance, you must run the database in close proximity to the {product-title-short} Central services.
+
+For more information about the supported platforms, see the Red Hat Knowledgebase solution link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix].
```

Comment:
```suggestion
[id="bring-your-own-database-for-central-is-now-generally-available_{context}"]
=== Bring your own database for RHACS Central is now generally available

With {product-title-short} 4.4, you can now use your own PostgreSQL-compatible database with {product-title-short} Central, allowing you to deploy it within or outside the cluster. Whether on bare metal, virtual machines, or as a cloud-hosted service, you can tailor the deployment to your specific requirements. To ensure optimal performance, you must run the database close to the {product-title-short} Central services.

For more information, see xref:../../installing/installing_ocp/install-central-ocp.adoc#install-central-operator-external-db_install-central-ocp[Installing Central with an external database using the Operator method].
```

**nitpick:** I've updated the language to match waht we used in previous releases and added a link to the docs.